app/views/groups/new.html.erb

```diff
@@ -0,0 +1,51 @@
+<% if @group.errors.any? %>
+  <div id="error_explanation">
+
+    <ul>
+      <% @group.errors.full_messages.each do |msg| %>
+        <li class="alert-danger"><%= msg %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+<main class="body-container">
+  <%= form_with model: @group, url: user_groups_path, class: "container", local: true, html: {multipart: true} do |f| %>
+
+    <div class="field">
+      <%= f.text_field :name, placeholder: "Category name", class: 'form-control' %>
+    </div>
+
+    <div class="field">
+      <%= f.file_field :icon, id: "user_avatar" %>
+      <br>
+      <div class="avatar-preview-container">
+        <% if @group.icon.attached? %>
+          <%= image_tag @group.icon, class: "avatar" %>
+        <% else %>
+          <%= image_tag 'default_product_image.webp', class: "avatar" %>
+        <% end %>
+      </div>
+    </div>
+
+
+    <div class="actions">
+      <%= f.submit class: 'btn btn-primary', data: { turbo: false } %>
+    </div>
+  <% end %>
+
+  <%= link_to 'Back', user_groups_path, class: 'gray-btn' %>
```

Comment:
Great design @ONJoseph 🤝

- [ ] As requested by the previous reviewers, kindly implement the back button on most of your pages so that your design can replicate the design provided.
- Your design
![back-button](https://github.com/ONJoseph/BudgetApp/assets/63917725/e2af2a60-3dad-4307-8fd9-2217c60f5da5)
- Given design
![given back button](https://github.com/ONJoseph/BudgetApp/assets/63917725/8ebf7512-b573-4995-bc6e-d25751e74c45)

Reply:
@1GORDON , Thanks a lot for the review. I have worked hard to implemented the changes. Please kindly approve to enable me to progress in the program and not repeat the week a second time. Thanks for the support!

Reply:
- Well resolved 👍🏽 
- [ ] [OPTIONAL] However, let's remove the menu icon and ensure the back button is rightly placed at the edge of the screen as indicated in the template provided.
<img width="459" alt="Screenshot 2023-05-13 at 00 21 34" src="https://github.com/ONJoseph/BudgetApp/assets/58692590/2bc03d59-1b82-4ca5-8ef6-b979abe5bfb6">